src/main/java/nl/haris/backendspringboottechiteasymodel/controllers/TelevisionController.java

```diff
@@ -44,55 +51,79 @@ public ResponseEntity<Television> getTelevision(@PathVariable Long id) {
     @GetMapping
     public ResponseEntity<List<Television>> getAllTelevisions(@RequestParam(name = "brand",
             required = false) String brand) {
+
+        List<Television> filteredTelevisions;
```

Comment:
Waarom maak je hier 2 lijsten aan? Of je er nou "findbyBrand" inzet of "findAll", maakt niet uit. Beide is gewoon een `List<Television>` en in beide gevallen kun je `ResponseEntity.ok(televisionList)` returnen.